dotfiles.py

```diff
@@ -1,29 +1,52 @@
 #!/usr/bin/env python3.8
-"""Module docstring."""
+
 import filecmp
+from collections import namedtuple
```

Comment:
I don't think you need to import `namedtuple` from `collections` if you use the `NamedTuple` from `typing`.

Reply:
D'oh!